app/components/qualification_subject_component.rb

```diff
@@ -0,0 +1,13 @@
+class QualificationSubjectComponent < ViewComponent::Base
+  def initialize(qualification:)
+    @qualification = qualification
+  end
+
+  def subject
+    @qualification.subject&.titleize
```

Comment:
Do we allow this to be nil? We validate :subject on GcseQualificationTypeForm, OtherQualificationForm and DegreeSubjectForm

Reply:
We do, yes. A candidate can abandon the 'add degree' flow partway through, meaning this table may need to render a partially-completed degree.